### PR TITLE
Update streamr-client to v3

### PR DESCRIPTION
**Pull request by mondoreale**
For whatever reason running canvases on 2.2.7 stopped working. Looks like with v3 it's fine again.

### CODE REVIEW

**Comment by fonty1** [comment]
@harbu is there any breaking changes from v2 -> v3 we should look out for? And is it production ready?

**Comment by mondoreale** [comment]
cc @mirpo @mthambipillai

**Comment by mirpo** [comment]
Yep, it's production ready. But in version 3 we still have problems with resends. Next week I'll release new version with (lets say) more stable resends :)

**Comment by mondoreale** [comment]
Thanks @mirpo. I'mma merge it in a minute (unless someone stops me!). 🙌

**Comment by mirpo** [comment]
wait for https://github.com/streamr-dev/streamr-platform/pull/839 to check tests all tests first

**Comment by fonty1** [comment]
@mondoreale you can rebase now, and with any luck you should see 🟢

**Comment by mirpo** [comment]
@mondoreale I just published 3.0.1 with more stable resends, please update

**Comment by mondoreale** [comment]
Will update shortly. Thank you guys!

File: app/package.json [hunk 1]
```diff
@@ -246,7 +246,7 @@
     "storybook-addon-jsx": "^5.4.0",
     "storybook-chromatic": "^3.4.1",
     "storybook-react-router": "^1.0.1",
-    "streamr-client": "^2.2.7",
+    "streamr-client": "^3.0.1",
```
Comment (mondoreale) [inline app/package.json]: v3.0.1 it is!

**Pull request merged**